### PR TITLE
fix some ineffassign to improve GoReport Result.

### PR DESCRIPTION
**Pull request by yangwenmai**

### CODE REVIEW

File: table/namespace_classifier_test.go [hunk 1]
```diff
@@ -154,14 +154,16 @@ func (s *testTableNamespaceSuite) TestNamespaceOperation(c *C) {
 
 	// Add tableID
 	err = tableClassifier.AddNamespaceTableID("test1", 1)
-	namespaces = tableClassifier.GetNamespaces()
 	c.Assert(err, IsNil)
+	namespaces = tableClassifier.GetNamespaces()
+	c.Assert(len(namespaces), Equals, 1)
```
Comment (siddontang) [inline table/namespace_classifier_test.go]: can we use `c.Assert(namespaces, HasLen, 1)` here?

**Comment by siddontang** [comment]
Rest LGTM

PTAL @disksing

**Comment by disksing** [comment]
Thanks @yangwenmai , please check out why CI is failed.

**Comment by yangwenmai** [comment]
Yes, I'm fixing.

File: table/namespace_classifier_test.go (hunk 1 above)
Comment (yangwenmai) [reply table/namespace_classifier_test.go]: OK.

**Comment by disksing** [comment]
Thanks again. @yangwenmai

**Pull request merged**